tests_integration/test_ssh.py

```diff
@@ -41,6 +42,48 @@ def proxy_cli(*args, **kwargs) -> subprocess.CompletedProcess:
     )
 
 
+def stdio_cli(*args, **kwargs) -> subprocess.CompletedProcess:
+    """Execute the proxy cli command using stdin/out forwarding"""
+    if not shutil.which("ssh"):
+        pytest.fail(
+            "ssh client not found. Please make sure the 'ssh' client is included in your PATH variable"
+        )
+
+    return subprocess.run(
+        [
+            "ssh",
+            "-o",
```

Comment:
Could you please add the following ssh option to the command, otherwise ssh will prompt for the fingerprint when running automated tests.

```python
"-o",
"StrictHostKeyChecking=no",
```